File: user-service/src/main/java/rs/edu/raf/dto/CompanyDTO.java
package rs.edu.raf.dto;

public record CompanyDTO(
        Long id,
        String phoneNumber,
        String address,
        String name,
        String accountNumbers,
        String faxNumber,
        Integer TIN,
        Integer registrationNumber,
        Integer businessActivityCode,
        Integer registryNumber) {
}

File: user-service-kotlin/src/main/kotlin/org/example/userservicekotlin/dto/CompanyDTO.kt
package org.example.userservicekotlin.dto

data class CompanyDTO(
    val id: Long?,
    val phoneNumber: String,
    val address: String,
    val name: String,
    val accountNumbers: String?,
    val faxNumber: String,
    val TIN: Int,
    val registrationNumber: Int,
    val businessActivityCode: Int,
    val registryNumber: Int
)
